guides/common/modules/con_organizations-and-locations-in-foreman.adoc

```diff
@@ -0,0 +1,15 @@
+[id="Organizations-and-Locations-in-{ProjectNameID}_{context}"]
+= Organizations and locations in {ProjectName}
+
+On the {ProjectServer}, you can define multiple _organizations_ and _locations_ to help organize content, managed hosts, and configurations.
+
+Organizations::
+Organizations typically represent different business units, departments, or teams, such as Finance, Marketing, or Web Development Team.
```

Comment:
Do we want/need some emphasis for the examples? I'm leaning towards italics.

Reply:
IBM Style has a good section on when (not) to add emphasis... but it's not really conclusive for this particular situation. Anyway, let's try it. Applied here and elsewhere.